scripts/setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# INSTRUCTIONS:
+# This script will move the pre-push hook from script folder to
+# the .git/hooks folder
+#
+# Run the script from the oppia-android root folder:
+#
+#   bash scripts/setup.sh
+#
+# NOTE: this script should be run once after the initial codebase setup
+
+# Move file from script folder to .git/hooks folder
+cp scripts/pre-push.sh .git/hooks/pre-push
+
+# Download ktlint
+KTLINT="0.37.0"
+echo Using Ktlint $KTLINT
+cd ..
+mkdir -p oppia-android-tools
+cd oppia-android-tools
```

Comment:
@BenHenning Let me know if there is any short hand for this

```console
cd ..
mkdir -p oppia-android-tools
cd oppia-android-tools
```

Reply:
Best I can come up with is:

```bash
mkdir -p ../oppia-android-tools
cd ../oppia-android-tools
```

(this might be slightly better since it's keeping the pwd within a directory that we manage)

Reply:
done
resolving on behalf of Ben